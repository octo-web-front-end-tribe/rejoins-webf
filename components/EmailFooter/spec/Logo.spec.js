import Vue from 'vue'
import LogoWebF from '@/components/EmailFooter/Logo'
import configuration from '@/contents/config'

describe('LogoWebF Component', () => {
    let $mounted

    const Constructor = Vue.extend(LogoWebF)

    beforeEach(() => {
        $mounted = new Constructor({
            propsData: {
                driveImageID: configuration.logoWebF.linkID
            }
        }).$mount()
    })

    test('snapshot', () => {
        let $html = $mounted.$el.outerHTML
        expect($html).toMatchSnapshot()
    })
})