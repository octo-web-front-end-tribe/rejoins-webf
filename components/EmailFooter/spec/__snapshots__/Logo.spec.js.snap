// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoWebF Component snapshot 1`] = `"<img src=\\"https://drive.google.com/uc?id=13BzL0ExgsfvDpbZsTQe4qq15XS1GS0_4\\" style=\\"width: 100px;\\">"`;
